script/Arcueid/Day 3/03_00_ARC03_6.txt

```diff
@@ -96,12 +97,12 @@ There's nothing particularly wrong with my body―――
 [sha:319bb7a767d7474c7081fc1dc08e4d581c4e026f]{
 -- Page 277, Offset 4761.
 -- そう、なんで。
-Yes, why?
+Yeah... why?
 }
 [sha:0ebf3d7d85a94a6154755b01d957b321b0a30ee9]{
 -- Page 277, Offset 4762.
 -- 自分は、こんなところで眠っているんだ―――？
-Why am I sleeping in a place like this―――?
+Why am I back here, sleeping in my own bed?
 }
 [sha:c337ca4ee7ff6930288fc0590233ff238f1edade]{
 -- Page 278, Offset 4763.
```

Comment:
There's a person... that I... ki―――

Reply:
Hmmm I know it says 'that woman' but it really does feel like the more natural sentence here to me

Reply:
just get it off my mind